ui/src/app/base/components/ActionForm/ActionForm.test.tsx

```diff
@@ -71,95 +77,12 @@ describe("ActionForm", () => {
     expect(wrapper.find("ActionButton").prop("disabled")).toBe(true);
   });
 
-  it("runs clearHeaderContent function when processing complete", () => {
```

Comment:
I've moved this out of `ActionForm` so it's not responsible for closing the form - this is to remove a workaround in the clone form, which doesn't close when it's complete (it shows a results page)